Livecheckables/openssh.rb

```diff
@@ -0,0 +1,6 @@
+class Openssh
+  livecheck do
+    url "https://ftp.openbsd.org/pub/OpenBSD/OpenSSH/portable/"
+    regex(/href=.*?openssh-v?(\d+(?:\.\d+)+(?:p\d+))\.t/i)
```

Comment:
```suggestion
    regex(/href=.*?openssh-v?(\d+(?:\.\d+)+(?:p\d+)?)\.t/i)
```

Though the current file names all contain something like `p1` at the end, we may want to make it optional in case we ever encounter a version without it. Looks good outside of this.